llvm/include/llvm/SandboxIR/SandboxIRValues.def

```diff
@@ -33,45 +33,46 @@ DEF_USER(ConstantInt, ConstantInt)
 #define OPCODES(...)
 #endif
 // clang-format off
-//       ClassID,        Opcode(s),         Class
-DEF_INSTR(Opaque,        OP(Opaque),        OpaqueInst)
+//        ClassID,        Opcode(s),         Class
+DEF_INSTR(Opaque,         OP(Opaque),        OpaqueInst)
 DEF_INSTR(ExtractElement, OP(ExtractElement), ExtractElementInst)
-DEF_INSTR(InsertElement, OP(InsertElement), InsertElementInst)
-DEF_INSTR(Select,        OP(Select),        SelectInst)
-DEF_INSTR(Br,            OP(Br),            BranchInst)
-DEF_INSTR(Load,          OP(Load),          LoadInst)
-DEF_INSTR(Store,         OP(Store),         StoreInst)
-DEF_INSTR(Ret,           OP(Ret),           ReturnInst)
-DEF_INSTR(Call,          OP(Call),          CallInst)
-DEF_INSTR(Invoke,        OP(Invoke),        InvokeInst)
-DEF_INSTR(CallBr,        OP(CallBr),        CallBrInst)
-DEF_INSTR(GetElementPtr, OP(GetElementPtr), GetElementPtrInst)
-DEF_INSTR(Switch,        OP(Switch),        SwitchInst)
-DEF_INSTR(UnOp,          OPCODES( \
-                         OP(FNeg) \
-                         ),                 UnaryOperator)
+DEF_INSTR(InsertElement,  OP(InsertElement), InsertElementInst)
+DEF_INSTR(ShuffleVector,  OP(ShuffleVector),  ShuffleVectorInst)
```

Comment:
nit: Align `ShuffleVectorInst` with the rest ?

Reply:
Done.